tasks/geonode.yml

```diff
@@ -233,11 +274,13 @@
 - name: create the db datastore in geoserver
   command: "{{virtualenv_dir}}/{{app_name}}/bin/python {{app_code_dir}}/{{app_name}}/create_db_store.py"
 
-- name: Copy super user fixture in place
-  template: src=sample_admin.json.j2 dest={{app_code_dir}}/{{app_name}}/sample_admin.json
+- name: template superusers fixture in place
+  template:
+    src: superusers.json.j2
+    dest: "~/superusers.json"
```

Comment:
superusers.json is a template.  If it's in the  ~/<app_name> folder than it will create local changes in the git repo and prevent git pulls.  We could delete it after it's imported, but thought it was okay to leave it.